server.js

```diff
@@ -15,27 +17,36 @@ app.get('/api/v1/tasks', (req, res) => {
   res.status(200).json({ tasks: app.locals.tasks });
 });
 
+app.get('/api/v1/savedtasks', (req, res) => {
+  res.status(200).json(app.locals.savedTasks);
+});
+
 app.get('/api/v1/tasks/:category', (req, res) => {
-  let categoryKeys = Object.keys(app.locals.tasks)
-  let arr = []
   let category = req.params.category
-  categoryKeys.forEach(key => {
-    app.locals.tasks[key].forEach((task) => {
-      let lowercase = task.category
-      let findWhiteSpace = lowercase.replace(/ /g, '')
-      lowercase = findWhiteSpace.toLowerCase()
-      if (lowercase === category) {
-        arr.push(task)
-      }
-    })
-  })
-  if (arr.length !== 0) {
-    res.status(200).json(arr)
+  if (req.params.category === 'mentalcare'){
+    category = 'mentalCare'
+  }
+  const foundTasks = app.locals.tasks[category.toLowerCase()]
+
+  if (foundTasks.length) {
+    res.status(200).json(foundTasks)
   } else {
     res.sendStatus(404)
   }
-})
+});
 
+app.post('/api/v1/savedtasks', (req, res) => {
+  app.locals.savedTasks.push(req.body);
```

Comment:
Hey! Is there room for error handling here? 
Would the request body always be as you expect it to be?

Reply:
Great question! That's what I was wondering too, but the only way to save a task is on the task page so there's no way to save the wrong thing.